src/components/Gradebook/GradebookFilters/index.jsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable react/sort-comp, import/no-named-as-default */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Collapsible, Form } from '@edx/paragon';
+
+import * as filterActions from 'data/actions/filters';
+
+import AssignmentTypeFilter from './AssignmentTypeFilter';
+import AssignmentFilter from './AssignmentFilter';
+import AssignmentGradeFilter from './AssignmentGradeFilter';
+import CourseGradeFilter from './CourseGradeFilter';
+import StudentGroupsFilter from './StudentGroupsFilter';
+
+export class GradebookFilters extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      includeCourseRoleMembers: this.props.includeCourseRoleMembers,
+    };
+    this.handleIncludeTeamMembersChange = this.handleIncludeTeamMembersChange.bind(this);
+  }
+
+  handleIncludeTeamMembersChange(includeCourseRoleMembers) {
+    this.setState({ includeCourseRoleMembers });
```

Comment:
I don't totally follow. Are we keeping this in both this component's state and redux state? why?

Reply:
to make react controlled component happy.  Doesn't like being controlled by external value when it also allows toggling of that value.